### PR TITLE
Remove links to Gists

### PR DESCRIPTION
**Pull request by jekyllbot**
Following #6744 

```text
Remove links to Gists
```

### CODE REVIEW

File: docs/_docs/plugins.md [hunk 1]
```diff
@@ -910,7 +877,7 @@ You can find a few useful plugins at the following locations:
   plugin to generate publication lists from [BibSonomy](https://www.bibsonomy.org/).
 - [github-cards](https://github.com/edward-shen/github-cards): Creates styleable Github cards for your Github projects.
 - [disqus-for-jekyll](https://github.com/kacperduras/disqus-for-jekyll): A Jekyll plugin to view the comments powered by Disqus.
-- [jekyll-html](https://github.com/kacperduras/jekyll-html): A Jekyll plugin to use HTML tags in Jekyll pages, posts and collections. 
+- [jekyll-html](https://github.com/kacperduras/jekyll-html): A Jekyll plugin to use HTML tags in Jekyll pages, posts and collections.
```
Comment (pathawks) [inline docs/_docs/plugins.md]: Can we avoid modifying this line (because history/blame)?

(I feel the pain of automatically trimming whitespace 😁)

Comment (DirtyF) [reply docs/_docs/plugins.md]: I configured Atom to fix this automatically, that page should be history soon once we ship a decent version of https://jekyll.github.io/directory/

Comment (DirtyF) [reply docs/_docs/plugins.md]: done

**Comment by mattr-** [comment]
Why remove the gists? Because they're not gem based plugins? Did we deprecate non-gem based plugins and I missed it?

**Comment by DirtyF** [comment]
@mattr, Most of those gists are unmaintained and we have no way of checking they still work with latest Jekyll. We should encourage plugins authors to release gems. That's the whole point or moving all the plugins from this page and continue the work on https://jekyll.github.io/directory/

**Comment by mattr-** [comment]
jekyllbot: merge +site

**Comment by mattr-** [comment]
@jekyllbot: merge +site

**Pull request merged**